### ISSUE TITLE
Update dependencies to use new mt-downloader version

### ISSUE THREAD

**Issue by leeroybrun** [leeroybrun/node-mt-files-downloader#5]
We should update the dependency and check that the module is still working.

**Comment by philipgiuliani** [comment]
Would be awesome if you could do this!

**Comment by leeroybrun** [comment]
Yup! Unfortunately I'm quite busy these times because I'm currently finishing my Bachelor degree.

But will try to do this as soon as possible :-)

**Comment by philipgiuliani** [comment]
Have you looked at the fork of @zeltice ? It looks really good and he wrote his [own version of the mt-downloader](https://github.com/zeltice/zeltice-mt-downloader) which seem to be much better for me!

https://github.com/zeltice/node-mt-files-downloader
